libs/chain-signatures/contract/src/lib.rs

```diff
@@ -725,46 +725,21 @@ impl VersionedMpcContract {
         if let Some(legacy_contract_state::VersionedMpcContract::V1(state)) =
             env::state_read::<legacy_contract_state::VersionedMpcContract>()
         {
-            // migrate config
             let mut config = Config::default();
             config.request_timeout_blocks = state.config.request_timeout_blocks;
             config.max_num_requests_to_remove = state.config.max_num_requests_to_remove;
```

Comment:
I don't think we need these anymore since #296 got merged.

Reply:
Oh yeah, let me remove those too.